### PR TITLE
Added postman Collection

### PR DESCRIPTION
**Pull request by ankitpws**
Added postman Collection

### CODE REVIEW

File: src/api-doc/MentorED-Users.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,909 @@
...
+									"value": "timeout=5"
+								}
+							],
+							"cookie": [],
+							"body": "{\n    \"responseCode\": \"CLIENT_ERROR\",\n    \"message\": \"User already exists.\",\n    \"error\": []\n}"
+						}
+					]
+				},
+				{
+					"name": "Create Account Mentor",
```
Comment (rakeshSgr) [inline src/api-doc/MentorED-Users.postman_collection.json:535]: @ankitpws keep it Create Account

File: src/api-doc/MentorED-Users.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,909 @@
...
+		"name": "MentorED-Users",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "7997930"
+	},
+	"item": [
+		{
+			"name": "Profile",
+			"item": [
+				{
+					"name": "Profile Details (Mentor + Mentee)",
```
Comment (rakeshSgr) [inline src/api-doc/MentorED-Users.postman_collection.json:13]: @ankitpws change to "Profile Details"

File: src/api-doc/MentorED-Users.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,909 @@
...
+						"url": {
+							"raw": "{{UserDevBaseUrl}}user/v1/profile/details/62832531a05cbd57b273aebb",
+							"host": ["{{UserDevBaseUrl}}user"],
+							"path": ["v1", "profile", "details", "62832531a05cbd57b273aebb"]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Profile Update (Mentor + Mentee)",
```
Comment (rakeshSgr) [inline src/api-doc/MentorED-Users.postman_collection.json:39]: @ankitpws change to "Profile Update"

File: src/api-doc/MentorED-Users.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,909 @@
...
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "mentors",
+			"item": [
+				{
+					"name": "Mentor Directory",
```
Comment (rakeshSgr) [inline src/api-doc/MentorED-Users.postman_collection.json:321]: @ankitpws change to Mentor list

Comment (ankitpws) [reply src/api-doc/MentorED-Users.postman_collection.json:321]: yes done but in mentoring services

**Pull request merged**